appservice/src/disconnectRepo.ts

```diff
@@ -0,0 +1,22 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { SiteSourceControl } from 'azure-arm-website/lib/models';
+import { MessageItem } from 'vscode';
+import { IActionContext } from 'vscode-azureextensionui';
+import { editScmType } from './';
+import { ext } from './extensionVariables';
+import { localize } from './localize';
+import { ScmType } from './ScmType';
+import { DeploymentsTreeItem } from './tree/DeploymentsTreeItem';
+
+export async function disconnectRepo(context: IActionContext, node: DeploymentsTreeItem): Promise<void> {
```

Comment:
I would make the `node` parameter less explicit. It should be the same type as the `node` parameter needed for `editScmType`

Reply:
Done